min.py

```diff
@@ -135,6 +136,37 @@ def get_min_scipy_version(min_python, min_numpy):
     return df.SciPy_version
 
 
+def find_pkg_mismatches(pkgs, fnames):
+    """
+    Determine if any package version mismatches
+    """
+    pkg_mismatches = []
+
+    for pkg_name, pkg_version in pkgs.items():
+        for fname in fnames:
+            with open(fname, "r") as file:
+                for line_num, line in enumerate(file, start=1):
+                    l = line.strip().replace(" ", "").lower()
+                    matches = re.search(
+                        rf"""
+                            {pkg_name}  # Package name
+                            [=><:"\'\[\]]+  # Zero or more special characters
+                            (\d+\.\d+[\.0-9]*)  # Capture "version" in `matches`
+                          """,
+                        l,
+                        re.VERBOSE,  # Ignores all whitespace in pattern
+                    )
+                    if matches is not None:
+                        version = matches.groups()[0]
+                        if version != pkg_version:
+                            pkg_mismatches.append(
+                                f'Package Mismatch Found: "{pkg_name}" "{version}" '
+                                f"in {fname}:{line_num}"
```

Comment:
Will we need to append a `\n` (newline) to the end of the string?

Reply:
I don't believe so! The split between line 163 and 164 was to satisfy flake8's length concern! 

It seems to append correctly!

Reply:
Oh, I meant at the end of the combined strings. This way, they can be printed on separate lines later below in `__main__`